experiments/modern/src/runtime/Button.js

```diff
@@ -0,0 +1,21 @@
+const GuiObject = require("./GuiObject");
+
+class Button extends GuiObject {
+  /**
+   * getClassName()
+   *
+   * Returns the class name for the object.
+   * @ret The class name.
+   */
+  static getClassName() {
+    return "Button";
+  }
+  init(newRoot) {
+    return null;
+  }
+  setXMLParam(param, value) {
+    return null;
+  }
```

Comment:
We should move this to `GuiObject` and remove it from this and all the other decedents of `GuiObject`